.github/ISSUE_TEMPLATE/NEW_RELEASE.md

```diff
@@ -35,12 +35,13 @@ Please do not remove items from the checklist
       and inserts the changelog into the tag description.
```

Comment:
When building/developing Kueue on a remote machine, it is easier to do things in a different order:
1. push the tag
2. create draft referencing the tag
3. use `gh release upload <tag> artifacts/*`